.github/workflows/benchmark_on_push.yml

```diff
@@ -49,7 +53,8 @@ jobs:
           HEAD_COMMIT=$(git rev-parse HEAD)
           echo $BASE_COMMIT | tee commits_to_compare.txt
           echo $HEAD_COMMIT | tee -a commits_to_compare.txt
-          asv run HASHFILE:commits_to_compare.txt --m "GitHubRunner" --show-stderr --strict -v
+          asv run HASHFILE:commits_to_compare.txt --m "GitHubRunner" --show-stderr -v
```

Comment:
Re: removal of `--strict`

Do we want this workflow to pass even if the benchmarks fail? `asv` exits with a non-zero return code only if `--strict` is present. If we remove this, the workflow will always pass.

Reply:
They deprecated the `--strict` command-line flag in `0.6` so the benchmarks will always fail if they are destined to ([changelog](https://asv.readthedocs.io/en/v0.6.1/changelog.html#id5))

Reply:
Oh nice! Thanks for the clarification!